### PR TITLE
Updated Filesystem Documentation: Handling Deeply Nested JSON

### PR DESCRIPTION
**Pull request by dat-a-man**
This PR updates the filesystem documentation to include instructions for handling deeply nested JSON files by enabling automatic directory creation.
Changes:

Documentation Update:
- Provided instructions for setting kwargs = '{"auto_mkdir": true}' in config.toml.
- Included steps for setting kwargs using environment variables.

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md [hunk 1]
```diff
@@ -371,6 +371,14 @@ To configure the layout and placeholders using `config.toml`, use the following
 layout = "{table_name}/{test_placeholder}/{YYYY}-{MM}-{DD}/{ddd}/{mm}/{load_id}.{file_id}.{ext}"
 extra_placeholders = { "test_placeholder" = "test_value" }
 current_datetime="2024-04-14T00:00:00"
+# for deeply nested layouts 
+kwargs = '{"auto_mkdir": true}'
+```
+
+To set using environment variables:
```
Comment (VioletM) [inline docs/website/docs/dlt-ecosystem/destinations/filesystem.md:378]: Let's move this to the tip above (line 362)

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md [hunk 2]
```diff
@@ -371,6 +371,14 @@ To configure the layout and placeholders using `config.toml`, use the following
 layout = "{table_name}/{test_placeholder}/{YYYY}-{MM}-{DD}/{ddd}/{mm}/{load_id}.{file_id}.{ext}"
 extra_placeholders = { "test_placeholder" = "test_value" }
 current_datetime="2024-04-14T00:00:00"
+# for deeply nested layouts
```
Comment (VioletM) [inline docs/website/docs/dlt-ecosystem/destinations/filesystem.md:374]:
```suggestion
# for automatic directory creation in the local filesystem
```

**Review by VioletM — CHANGES_REQUESTED** [review]

File: docs/website/docs/dlt-ecosystem/destinations/filesystem.md (hunk 2 above)
Comment (dat-a-man) [reply docs/website/docs/dlt-ecosystem/destinations/filesystem.md:374]: updated

**Comment by AstrakhantsevaAA** [comment]
@dat-a-man, you trying to merge changes from master to devel, you don;'t need to do that. Which branch did your branch originate from master or devel?

**Pull request closed without merging**